pkgs/tools/audio/google-music-scripts/default.nix

```diff
@@ -1,19 +1,36 @@
 { lib, python3 }:
 
-with python3.pkgs;
+let
+  py = python3.override {
```

Comment:
It would be nice if everywhere we did this, we used a composable form, like the one in https://github.com/NixOS/nixpkgs/issues/44426

As it stands, this wipes out any other overlays you have, so if you have something like an SSL cert fix for python `certifi`, your google-music-scripts pkg won't have it.

Reply:
since this is an application, and the main concern is just that we can do `google-music-scripts ... `, I don't think it's too much of a concern.

In general though, I agree, it would be nice if this was more composable

Reply:
also related: https://github.com/NixOS/nixpkgs/pull/91850